docs/sources/clients/promtail/scraping.md

```diff
@@ -422,7 +422,7 @@ scrape_configs:
 ```
 
 Only `api_token` and `zone_id` are required.
-Refer to the [Cloudfare](../configuration/#cloudflare) configuration section for details.
+Refer to the [Cloudfare](configuration/#cloudflare) configuration section for details.
```

Comment:
To prevent broken links in the future, one can use the `relref` tag:

```suggestion
Refer to the [Cloudfare]({{< relref "configuration.md#cloudflare" >}}) configuration section for details.
```